src/components/SideNavbar/SideNavbar.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useState } from 'react';
+import styled from 'styled-components';
+// eslint-disable-next-line sort-imports
+import SidenavMenu from './SidenavMenu';
+
+const Nav = styled.div`
+  width: 80px;
+  height: 1110px;
+  padding: 40px 24px 91px;
+
+  box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.15);
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const KeywordBlock = styled.div`
+  width: 32px;
+  height: 32px;
+  margin: 20px 0;
+  object-fit: contain;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const HistoryBlock = styled.div`
+  width: 32px;
+  height: 32px;
+  ${({ sideMenu }) => {
+    return sideMenu ? `margin :   289px 0 0 0 ` : `margin: 20px 0`;
+  }};
+  object-fit: contain;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const ChatBlock = styled.div`
+  width: 32px;
+  height: 32px;
+  ${({ sideMenu }) => {
+    return sideMenu ? `margin :   45px 0 0 0 ` : `margin: 20px 0`;
+  }};
+  object-fit: contain;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const SetBlock = styled.div`
+  width: 32px;
+  height: 32px;
+  ${({ sideMenu }) => {
+    return sideMenu ? `margin :   397px 0 0 0 ` : `margin: 20px 0`;
+  }};
+  object-fit: contain;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const OpenMenu = styled.div`
+  width: 26px;
+  height: 20px;
+  margin: 0 3px 60px;
+  object-fit: contain;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const MenuButton = styled.button`
+  width: 26px;
+  height: 20px;
+  cursor: pointer;
+  object-fit: contain;
+  border: 0;
+  display: flex;
+  background-color: #fff;
+  justify-content: center;
+  align-items: center;
+`;
+
+const NavContainer = styled.div`
+  width: 395px;
+  height: 1110px;
+  display: flex;
+`;
+
+const SideNavbar = () => {
+  const [sideMenu, setsideMenu] = useState(false);
+
+  const showSideMenu = () => {
+    setsideMenu((current) => !current);
+  };
+
+  return (
+    <NavContainer>
+      <Nav>
+        <OpenMenu>
+          <MenuButton type="button" onClick={() => showSideMenu()}>
```

Comment:
함수에 인자가 없으면 함수 그대로 넘겨줘도 됩니다.
의미를 잘 생각해보세요

```suggestion
          <MenuButton type="button" onClick={showSideMenu}>
```

Reply:
092ef69